application/aam-backend-service/src/main/kotlin/com/aamdigital/aambackendservice/reporting/reportcalculation/core/DefaultReportCalculator.kt

```diff
@@ -28,8 +34,15 @@ class DefaultReportCalculator(
                     return@flatMap Mono.error(InvalidArgumentException())
                 }
 
+                if (reportCalculation.args["from"] == reportCalculation.args["to"]) {
+                    reportCalculation.args.remove("to")
+                }
```

Comment:
doesn't that mean a report requested for a single day results in a calculation for an open ended time period?

Reply:
just "from" would lead to "everything from until ever"
"from" and "to" with the same date would lead to "all from that specific day"